cmd/scheduler/app/app.go

```diff
@@ -82,24 +78,25 @@ func Run() {
 				return serr
 			}
 
-			server := server.New(server.Options{
-				AppID:            appID,
-				HostAddress:      hostAddress,
-				ListenAddress:    opts.ListenAddress,
-				PlacementAddress: opts.PlacementAddress,
-				Mode:             modes.DaprMode(opts.Mode),
-				Port:             opts.Port,
-				Security:         secHandler,
+			server, err := server.New(server.Options{
+				Port:          opts.Port,
+				ListenAddress: opts.ListenAddress,
+				Mode:          modes.DaprMode(opts.Mode),
+				Security:      secHandler,
 
 				DataDir:                 opts.EtcdDataDir,
-				Id:                      opts.Id,
+				ReplicaTotal:            opts.ReplicaTotal,
```

Comment:
we use replicaCount in the charts. can we use that here too for consistency?
```suggestion
				ReplicaCount:            opts.ReplicaCount,
```